exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/metrics/ValueAtQuantileMarshaler.java

```diff
@@ -26,8 +26,8 @@
     return marshalers;
   }
 
-  private static ValueAtQuantileMarshaler create(ValueAtPercentile value) {
-    return new ValueAtQuantileMarshaler(value.getPercentile() / 100.0, value.getValue());
```

Comment:
Sorry I lied here is the other logic change. So no change for opencensus + OTLP. Some change for opencensus + prometheus but we probably have to do it.